prometheus.spec

```diff
@@ -0,0 +1,99 @@
+%define debug_package %{nil}
+
+Name:           prometheus
+Version:        %{version}
+Release:        1%{?dist}
+Summary:        Prometheus is a systems and service monitoring system. It collects metrics from configured targets at given intervals, evaluates rule expressions, displays the results, and can trigger alerts if some condition is observed to be true.
+Group:          System Environment/Daemons
+License:        See the LICENSE file at github.
+URL:            https://github.com/prometheus/prometheus
+Requires(pre):  /usr/sbin/useradd
+Requires:       systemd-units
```

Comment:
Which distro is this for? Systemd is not widespread enough to presume it's everywhere, and in the deb world anyway it'll be at least 2019 until that's the case. I imagine it's longer for the rpm world given RHEL.